nano/styles/_formatting.less

```diff
@@ -167,6 +167,10 @@
     display: none;
 }
 
+i.fancy {
```

Comment:
Honestly the way CSS determines priority drives me moderately mad... this shouldn't do anything, in theory, but it actually makes it so that this style takes priority over the .fa style because it also specifies the element.